CHANGELOG/v1.11.0-rc.0.md

```diff
@@ -0,0 +1,389 @@
+🚨 This is a RELEASE CANDIDATE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).
+<details>
+<summary>More details about the release</summary>
+
+:warning: **RELEASE CANDIDATE NOTES** :warning:
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.**X**.x -> v1.**X**.x
+- Workload Cluster: v1.**X**.x -> v1.**X**.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Highlights
+
+* REPLACE ME
```

Comment:
I think we've been skipping the highlights and deprecation warning sections, not sure if the criteria are different for a release candidate.

Reply:
You can add highlights from here: https://github.com/kubernetes-sigs/cluster-api/issues/12123#issuecomment-3089392632

Wrote that with Fabrizio last week. But didn't have time yet to go over all PRs in this release. I think we don't have deprecation warnings as of now

Reply:
Fixed

Reply:
@chandankumar4 I don't see a Highlights section in the current PR (not sure if you pushed all changes after the GitHub outage).

Reply:
@sbueringer said no worries, we can add the Highlights later directly in the GitHub release.